backend/spec/system/question_crud_spec.rb

```diff
@@ -24,10 +24,9 @@
           find('.option', text: question_type.name).click
         end
 
-        it 'shows all surveys' do
+        it 'expect to show admin and teacher surveys' do
           find('label', text: 'Survey').click
-          expect(page).to have_selector('.option', text: survey_admin.name)
-          expect(page).to have_selector('.option', text: survey_teacher.name)
+          expect(page).to have_selector('.option', text: survey_admin.name && survey_teacher.name)
```

Comment:
Did you know that this was possible? Just found out by myself xD So cool!

Reply:
I am not sure if this works, can you try something like `expect(page).to have_selector('.option', text: survey_admin.name && survey_teacher)` and tell me if it also pass?